tests/testthat/test-reprex.R

```diff
@@ -49,28 +48,28 @@ test_that("rlang::last_error() and last_trace() work", {
   )
   ret <- reprex(input = input, advertise = FALSE)
   m <- match("rlang::last_error()", ret)
-  expect_false(grepl("Error", ret[m + 1]))
+  expect_no_match(ret[m + 1], "Error")
   m <- match("rlang::last_trace()", ret)
-  expect_false(grepl("Error", ret[m + 1]))
+  expect_no_match(ret[m + 1], "Error")
 })
 
 test_that("reprex() works even if user uses fancy quotes", {
   skip_on_cran()
   withr::local_options(list(useFancyQuotes = TRUE))
   # use non-default venue to force some quoted yaml to be written
-  expect_error_free(reprex(1, venue = "R"))
+  expect_no_error(reprex(1, venue = "R"))
 })
 
 test_that("reprex() errors for an R crash, by default", {
-  # TODO: consider switching to expect_snapshot() after switch to 3e
+  # TODO: consider switching to expect_snapshot()
```

Comment:
Since this is shaping up to be ?the conclusion? of switching to testthat 3e, would you got ahead switch this to `expect_snapshot()`?

Likewise just below.

Reply:
I don't think the one below is worth adding as a snapshot as it spans many lines. I think it is fine as is.